aclk/aclk_util.c

```diff
@@ -125,9 +229,14 @@ static inline void generate_topic_cache(void)
  */
 const char *aclk_get_topic(enum aclk_topics topic)
 {
-    generate_topic_cache();
+    if (!aclk_topic_cache)
+        fatal("Topic cache not initialized");
```

Comment:
we should fail the aclk connection not the agent

Reply:
true, will change in a moment